chore: change wording and move periods outside of links (#1703)
clerk/clerk-docs: docs/components/user/user-button.mdx

description: Clerk's <UserButton /> component is used to render the familiar user button UI popularized by Google.
---

> ![The \<UserButton /> component renders the familiar user button UI popularized by Google](/docs/images/ui-components/user-button.svg).

Reviewer: ```suggestion
![The \<UserButton /> component renders the familiar user button UI popularized by Google.](/docs/images/ui-components/user-button.svg)
```

The `<UserButton />` component is used to render the familiar user button UI popularized by Google.

> Clerk is the only provider with multi-session support, allowing users to sign into multiple accounts at once and switch between them. For multi-session apps, the `<UserButton />` automatically supports instant account switching, without the need of a full page reload. For more information, you can see the [Multi-session applications guide](/docs/custom-flows/multi-session-applications#overview).

Reviewer: ```suggestion
Clerk is the only provider with multi-session support, allowing users to sign into multiple accounts at once and switch between them. For multi-session apps, the `<UserButton />` automatically supports instant account switching, without the need of a full page reload. For more information, see the [dedicated guide on multi-session apps](/docs/custom-flows/multi-session-applications#overview).
```
